active/social-media.md

```diff
@@ -0,0 +1,75 @@
+# Social Media Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official social media profiles on behalf of the Django Software Foundation.
+
+The goals of the group are:
+- Promote the use of Django among the World Wide Web development community.
+- Protect the framework's long-term viability by generating revenue through sponsor relationships/campaigns.
+- Assist the board in posting official news and updates from DSF.
+- Maintain and monitor automation in connection with posting DSF news and updates to social media channels.
+- Enforce the DSF's Code of Conduct on DSF's social media channels. This can include moderating comments that do not meet our CoC or blocking users that grossly or repeatedly violate the CoC.
```

Comment:
```suggestion
- Enforce the DSF's Code of Conduct on its social media channels.
  This can include moderating comments that do not meet our CoC or blocking users that grossly or repeatedly violate the CoC.
```